packages/compose/README.md

```diff
@@ -2,6 +2,52 @@
 
 The `compose` package is a collection of handy [Higher Order Components](https://facebook.github.io/react/docs/higher-order-components.html) (HOCs) you can use to wrap your WordPress components and provide some basic features like: state, instance id, pure...
 
+
+The **compose** function is an alias to flowRight from Lodash. It comes from functional programming world and allows to compose any number of functions. An example that illustrates it for two functions:
```

Comment:
Should it be placed under `Usage` section?

Reply:
I like it at the top since it is explaining what compose is, the top section is using examples to illustrate the definition, where the usage area is for specific use.